packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -180,12 +192,23 @@ export function getPropGetters<
       'aria-autocomplete': 'both',
       'aria-activedescendant':
         store.getState().isOpen && store.getState().activeItemId !== null
-          ? `${props.id}-item-${store.getState().activeItemId}`
+          ? getAutocompleteElementId(
+              props.id,
+              `item-${store.getState().activeItemId}`,
+              activeItem?.source
+            )
           : undefined,
-      'aria-controls': store.getState().isOpen ? `${props.id}-list` : undefined,
-      'aria-labelledby': `${props.id}-label`,
+      'aria-controls': store.getState().isOpen
+        ? store
+            .getState()
+            .collections.map(({ source }) =>
+              getAutocompleteElementId(props.id, 'list', source)
+            )
+            .join(' ')
+        : undefined,
```

Comment:
This gives a comprehensive value of all lists that are controlled by the input ([MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls#values)).